### PR TITLE
sdk_test: add environment variable tests

### PR DESCRIPTION
**Pull request by robhafner**
Add two environment variable tests which illustrate how an environment variable can be used in OPA to verify a JWT.

### CODE REVIEW

File: sdk/opa_test.go [hunk 1]
```diff
@@ -2150,6 +2151,150 @@ result := {
 	}
 }
 
+func TestOpaRuntimeEnvironmentVariableDefinedInOS(t *testing.T) {
+	os.Setenv("TOKEN_VERIFY_KEY", "B41BD5F462719C6D6118E673A2389")
```
Comment (srenatus) [inline sdk/opa_test.go:2155]: [nit] Let's use [`t.SetEnv`](https://pkg.go.dev/testing#T.Setenv)

**Review by srenatus — DISMISSED** [review]
Thanks for contributing! Just one nitpick re: os.Setenv vs t.SetEnv

**Comment by robhafner** [comment]
I updated the commit on my forked branch expecting it to update the PR, but it doesn't look like it did.  Is there something else that I need to do for the PR to be updated? 

https://github.com/open-policy-agent/opa/compare/main...robhafner:opa:envvars

**Comment by robhafner** [comment]
I think I was looking at an outdated compared. I think it looks correct now. 

https://github.com/open-policy-agent/opa/compare/c8217ece91e6f540fd2d449d0f068c81b9e0d153..ffed5e7de5b9498fcc5412e4f2f67c3f88856ef9

File: sdk/opa_test.go [hunk 2]
```diff
@@ -2150,6 +2150,149 @@ result := {
...
+	a := count(claims) > 0
+}
+			`,
+		}),
+	)
+
+	defer server.Stop()
+
+	testBundleResource := "/bundles/bundle.tar.gz"
+	testLabel := "a label"
```
Comment (srenatus) [inline sdk/opa_test.go:2254]: Is the label relevant to the test? I'd suggest removing it if not... 🤔 Sorry I didn't notice in the last round of reviewing

**Review by srenatus — DISMISSED** [review]

File: sdk/opa_test.go (hunk 2 above)
Comment (robhafner) [reply sdk/opa_test.go]: No, it's not relevant. I removed it from both tests and updated the PR. Thanks!

**Pull request merged**